plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/src/com/redhat/fabric8analytics/lsp/eclipse/ui/internal/WorkspaceFilesFinder.java

```diff
@@ -61,7 +64,8 @@ public static WorkspaceFilesFinder getInstance() {
 
 		for (Object o : selection.toList()) {
 			if (o instanceof IAdaptable) {
-				IFile pom = findPOM((IAdaptable) o);
+				IProject project = (IProject) ((IAdaptable) o).getAdapter(IProject.class);
```

Comment:
The selected item does not necessarily be a project, you can select pom.xml file direclty